src/modules/components/proposal/proposalActions/proposalActions/proposalActions.tsx

```diff
@@ -68,15 +75,30 @@ export const ProposalActions: React.FC<IProposalActionsProps> = (props) => {
                         {...web3Props}
                     />
                 ))}
-                <div className="mt-1 flex w-full flex-col gap-y-3 px-4 pb-4 md:flex-row-reverse md:px-6 md:pb-6">
+                {actions.length === 0 && (
+                    <EmptyState
+                        heading={copy.proposalActionsContainer.empty.heading}
+                        description={emptyStateDescription ?? copy.proposalActionsContainer.empty.description}
```

Comment:
I'd probably just mark the `emptyStateDescription` as required and let the app set the description for both use-cases, the add-actions and proposal-actions, otherwise it might be strange that a string is defined here and the other is defined on the app